Yosemite/Yosemite/Model/Updaters/ProductUpdater.swift

```diff
@@ -131,4 +132,67 @@ extension Product: ProductUpdater {
                        groupedProducts: groupedProducts,
                        menuOrder: menuOrder)
     }
+
+    public func shippingSettingsUpdated(weight: String?, dimensions: ProductDimensions, shippingClass: ProductShippingClass?) -> Product {
+        return Product(siteID: siteID,
+                       productID: productID,
+                       name: name,
+                       slug: slug,
+                       permalink: permalink,
+                       dateCreated: dateCreated,
+                       dateModified: dateModified,
+                       dateOnSaleStart: dateOnSaleStart,
+                       dateOnSaleEnd: dateOnSaleEnd,
+                       productTypeKey: productTypeKey,
+                       statusKey: statusKey,
+                       featured: featured,
+                       catalogVisibilityKey: catalogVisibilityKey,
+                       fullDescription: fullDescription,
+                       briefDescription: briefDescription,
+                       sku: sku,
+                       price: price,
+                       regularPrice: regularPrice,
+                       salePrice: salePrice,
+                       onSale: onSale,
+                       purchasable: purchasable,
+                       totalSales: totalSales,
+                       virtual: virtual,
+                       downloadable: downloadable,
+                       downloads: downloads,
+                       downloadLimit: downloadLimit,
+                       downloadExpiry: downloadExpiry,
+                       externalURL: externalURL,
+                       taxStatusKey: taxStatusKey,
+                       taxClass: taxClass,
+                       manageStock: manageStock,
+                       stockQuantity: stockQuantity,
+                       stockStatusKey: stockStatusKey,
+                       backordersKey: backordersKey,
+                       backordersAllowed: backordersAllowed,
+                       backordered: backordered,
+                       soldIndividually: soldIndividually,
+                       weight: weight,
+                       dimensions: dimensions,
+                       shippingRequired: shippingRequired,
+                       shippingTaxable: shippingTaxable,
+                       shippingClass: shippingClass?.slug,
+                       shippingClassID: shippingClass.flatMap({ Int($0.shippingClassID) }) ?? Int.min,
```

Comment:
This looks handy 🌟 Could be extracted into an extension later?

Reply:
this will be removed after updating `Int` remote IDs to `Int64` :)